### PR TITLE
Users/tashuk/amcheckerdocs

### PR DESCRIPTION
**Pull request by TarunKrShukla**
Updating Docs for ARM Checker

### CODE REVIEW

File: 03-Security-In-CICD/Readme.md [hunk 1]
```diff
@@ -376,3 +382,138 @@ conditions (e.g., back to back SVT failures) etc.
 
 [Back to top...](Readme.md#contents)
 
+# AzSK ARM Template Checker
+### Overview
+
+The ARM Template security check script runs a scan on your given ARM template to examine various conditions and configurations that need to be present in your ARMTemplate for secured resource deployment.
```
Comment (gamitt-msft) [inline 03-Security-In-CICD/Readme.md]: Make the capitalization of word "ARM Template" consistent.

File: 03-Security-In-CICD/Readme.md [hunk 2]
```diff
@@ -376,3 +382,138 @@ conditions (e.g., back to back SVT failures) etc.
...
+ with the path of your ARM Template
+```PowerShell
+Get-AzSKARMTemplateSecurityStatus –ARMTemplatePath <Path to ARM Template> -Preview 
+```
+The parameters used are:
+- ARMTemplatePath – Path to ARM Template file or folder
+> **Note**: This feature is in preview mode only. So, passing "–Preview" switch is mandatory. 
+[Back to top…](Readme.md#contents)
+
+### ARM Temaplate Checker - What is covered?
```
Comment (gamitt-msft) [inline 03-Security-In-CICD/Readme.md]: Spelling correction needed

File: 03-Security-In-CICD/Readme.md [hunk 3]
```diff
@@ -376,3 +382,138 @@ conditions (e.g., back to back SVT failures) etc.
...
+     - For the “Verify” controls, look at the expected value and description column in .CSV file to decide whether to consider the    control as "Passed" or not. 
+     - For the “Failed” controls, look at the .CSV file to get the supporting information like Expected value , Line No. and Resource path etc. 
+	 
+#### Scan multiple ARMTemplates :-	 
+To scan multiple ARMTemplates at a time you can pass folder path containing different ARM Template(s) to “–ARMTemplatePath” parameter in “Get-AzSKARMTemplateSecurityStatus” cmdlet.
+e.g. :
+```PowerShell
+ Get-AzSKARMTemplateSecurityStatus  –ARMTemplatePath "D:\DSRE\TestARMChecker\" –Preview [-Recurse]
+```
+> **Note**: You need to pass “-Recurse” switch in cmdlet if you want to scan ARM Temaplates in the specified location and in all child folders of the location.
```
Comment (gamitt-msft) [inline 03-Security-In-CICD/Readme.md]: Spelling correction "Temaplates"

File: 03-Security-In-CICD/Readme.md [hunk 4]
```diff
@@ -376,3 +382,138 @@ conditions (e.g., back to back SVT failures) etc.
...
+> **Note:** The VSTS dialog doesn't provide a good visual indication but the task does 
+get added when you click "Add" once!
+
+![03_Task_Catalog](../Images/03_Task_Catalog_ARM.JPG)
+
+**Step-3:** Specify the input parameters for the ARM Checker task.
+The "AzSK ARM Template Checker" task starts showing in the "Run on Agent" list and displays some configuration inputs that are required for the task to run. These are none other than the familiar options we have been specifying while running the AzSK ARM Template Checker manually - you can specify the target ARM Template file path or a folder path based on your requirment.
+
+Along with input parameter, you can check for below options
+<br/>**Recurse:** Switch this if you want to scan ARM Temaplates in the specified location and in all child folders of the location.
```
Comment (gamitt-msft) [inline 03-Security-In-CICD/Readme.md]: Spelling correction "Temaplates"

**Comment by TarunKrShukla** [comment]
Corrected spelling mistakes

**Pull request merged**